### ISSUE TITLE
Resonite still accepts controller input when not focused

### ISSUE THREAD

**Issue by VRBat** [Yellow-Dog-Man/Resonite-Issues#1675]
### Describe the bug?

When a controller is connected, Resonite will still accept controller inputs when the window is not focused. 

### To Reproduce

1. Connect controller
2. Focus out of resonite
3. Use controller inputs 

### Expected behavior

Controller inputs should **not** be accepted when Resonite is **not** focused.

### Resonite Version Number

2024.4.3.1170

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Desktop

### Log Files

[BATTOP - 2024.4.3.1170 - 2024-04-12 18_36_53.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/14964505/BATTOP.-.2024.4.3.1170.-.2024-04-12.18_36_53.log)

### Reporters

vrbat (June)
tatsukimiero (Tatsu)

**Comment by EmergencyTemporalShift** [comment]
I’d like to acknowledge that at least in VR, this is preferred behavior. I use desktop plus and often have other windows selected. Sometimes I am typing or moving the mouse, and I don’t wish to click back on the game every time I do so.

**Comment by Frooxius** [comment]
Yes, this it not actually a bug. There's actually a number of cases where you want the gaming input to be active, even when the window is not focused - if you're in VR like @EmergencyTemporalShift mentioned or even in desktop, you might be using another app, but have gamepad at hand to still control the game without needing focus.

What we'll probably do is make this configurable - add a toggle for whether the gamepad should be active or not when the app doesn't have a focus.

**Comment by VRBat** [comment]
I understand now! A toggle would indeed be very useful, good timing too since the settings rework is in progress
